shared/models/collections/collection.model.ts

```diff
@@ -18,7 +16,7 @@ export interface Collection extends Document {
 /* Helpers for building queries */
 export interface WorkInfo {
     readonly _id: string;
-    readonly author: string | AuthorInfo;
```

Comment:
This is one example of the Mongoose thing I was talking about. Now if we try to a do a query on `"author:" "somevalue"`, it'll complain. It's not actually a problem, it's just that the types library can't seem to model this correctly. The solution is just turning off the type checker for that one line.